include/circt/Dialect/Ibis/IbisTypes.td

```diff
@@ -14,18 +14,18 @@ include "mlir/IR/AttrTypeBase.td"
 
 class IbisTypeDef<string name> : TypeDef<IbisDialect, name> { }
 
-def ClassRefType : IbisTypeDef<"ClassRef"> {
-  let mnemonic = "classref";
-  let parameters = (ins OptionalParameter<"mlir::FlatSymbolRefAttr">:$classRef);
-  let assemblyFormat = "(`<` $classRef^  `>`)?";
+def ScopeRefType : IbisTypeDef<"ScopeRef"> {
+  let mnemonic = "scoperef";
+  let parameters = (ins OptionalParameter<"mlir::FlatSymbolRefAttr">:$scopeRef);
```

Comment:
uber-nit: by convention, the description comes before parameters and assemblyFormat.

Reply:
agreed